modules/visits/src/pages/Recommended-schools.js

```diff
@@ -20,7 +23,8 @@ import {
 } from "native-base";
 import RecommendedVisitsCard from "../components/RecommendedVisitsCard";
 import MySchoolsCard from "../components/MySchoolsCard";
-
+import colorTheme from "../colorTheme";
+const colors0 = overrideColorTheme(colorTheme);
```

Comment:
Hey @reddyganesha15 can we import it as:

`const colors = overrideColorTheme(colorTheme);`

Reply:
Need to remove `0` from end.